src/test/kotlin/io/imotions/bson4k/encoder/BsonClassEncoderTest.kt

```diff
@@ -89,11 +100,22 @@ class BsonClassEncoderTest : StringSpec({
     }
 
     "Encode class with default nullables" {
+        val encodeDefaultsBson = Bson {
+            encodeDefaults = true
+        }
         val clazz = Wrapper2Null<String, String>(y = "hello")
-        val document = bson.encodeToBsonDocument(clazz)
+        val document = encodeDefaultsBson.encodeToBsonDocument(clazz)
             .also { println(it) }
 
         document["x"] shouldBe BsonNull()
         document["y"] shouldBe BsonString("hello")
     }
+
+    "Encode class without defaults" {
```

Comment:
Some ambiguously named tests. At first, I read this as "encode a class which has no defaults". But, you mean "Encode class with default values and encodeDefaults disabled"